octoprint_octolapse/render.py

```diff
@@ -25,7 +26,7 @@ def __init__(
         octoprintTimelapseFolder, ffmpegPath, threadCount,
         timeAdded=0, onRenderStart=None, onRenderFail=None,
         onRenderSuccess=None, onRenderComplete=None, onAfterSyncFail=None,
-        onAfterSycnSuccess=None, onComplete=None):
+        onAfterSycnSuccess=None, onComplete=None, printStartTime=None):
```

Comment:
We probably don't need the printStartTime anymore either if we do the cleaning in timelapse.py

Reply:
Sounds good